app/services/login/user_verifier.rb

```diff
@@ -119,6 +128,10 @@ def backing_idme_uuid_has_changed?
       backing_idme_uuid != user_verification.backing_idme_uuid
     end
 
+    def credential_attributes_digest_changed?
```

Comment:
It might be here... but I think we also need to make sure we are only updating the digest when we're at an loa3/ial2 equivalent, so we should block it from updating somewhere if we're not at a verified level